### PR TITLE
Add POST action to admin/functions/{name}/keys

### PR DESCRIPTION
**Pull request by balag0**
ARM Api - "POST api/sites/{name}[/slots/{slot}]/functions/{functionName}/listsecrets" maps to this route and expects POST action to be allowed.

### CODE REVIEW

**Comment by bala16** [comment]
This also matches the behavior in kudu - https://github.com/projectkudu/kudu/blob/26fc8b0f9e3d9d2bb2304950545da4c713dd2d81/Kudu.Services/Functions/FunctionController.cs#L130

File: src/WebJobs.Script.WebHost/Controllers/KeysController.cs [hunk 1]
```diff
@@ -38,6 +38,7 @@ public KeysController(IOptions<ScriptApplicationHostOptions> applicationOptions,
         }
 
         [HttpGet]
+        [HttpPost]
```
Comment (mathewc) [inline src/WebJobs.Script.WebHost/Controllers/KeysController.cs]: Need to add test. There's already a KeysControllerTests suite. Just add a POST test matching the existing GET test

Comment (mathewc) [reply src/WebJobs.Script.WebHost/Controllers/KeysController.cs]: I see now that those tests are direct method tests - not full http tests.

**Comment by fabiocav** [comment]
Closing this. Please reopen if you still feel this is needed.

**Pull request closed without merging**